src/context/context-provider.ts

```diff
@@ -211,6 +211,9 @@ export class BBContextProvider {
               } else if (itemCount > 1) {
                 // Let the user pick a context.
                 showPicker(args, destinations, resolve, reject);
+              } else if (args.invalidContextHandler) {
```

Comment:
Instead of passing a handler, my recommendation would be to pass a `disableRedirect` flag. This code can then check for that flag and reject the Promise if it's present. This would follow the pattern in `BBAuth.getToken()`.

Reply:
See https://github.com/blackbaud/auth-client/blob/aa00f216b09833fa4f863ae4335b7345822fb171/src/shared/csrf-xhr.ts#L144

Reply:
Great suggestion.  Let me know if the current implementation is not what you had in mind.